style.css

```diff
@@ -21,6 +21,7 @@ hr {
   width: 20.313rem;
 }
 
+/* mobile design  */
 @media only screen and (min-width: 375px) {
```

Comment:
- Hey, since this is a mobile-first approach, kindly remove this media query as your code won't break, Instead add a media query from 768px to implement the design for the desktop version as you've currently not made any changes to your design for the desktop version.
- The works and about section is still not yet having any effect for the desktop version. Please, check the Figma template for your design to implement the desktop design as much as you can. After that, you can submit your project for a re-review.

Reply:
@zieeco i don't understand what you mean

Reply:
can you explain in more detail

Reply:
@Shahid-Bagwan, I'm sorry for the late reply. Your design currently is not visually close to the desktop design from that of the Figma design. You need to improve your design.

Reply:
@zieeco can you be more specific??, because i have checked it and am not able to find differences..

Reply:
@Shahid-Bagwan, You've made a good change now to your code and the design is better now. Try fixing the issue with the broken image on your headline section then you can submit it for a re-review and another CR would review your project.